code/game/objects/structures/transit_tubes/transit_tube_pod.dm

```diff
@@ -9,9 +9,9 @@
 	var/datum/gas_mixture/air_contents = new()
 
 
-/obj/structure/transit_tube_pod/New(loc)
-	..()
-	air_contents.assert_gases("o2", "n2")
+/obj/structure/transit_tube_pod/Initialize()
+	. = ..()
+	air_contents.add_gases("o2", "n2")
```

Comment:
still wondering why you changed it from assert_gases?

Reply:
see comment above